APMrover2/commands_logic.cpp

```diff
@@ -92,6 +92,12 @@ bool ModeAuto::start_command(const AP_Mission::Mission_Command& cmd)
         }
         break;
 
+    case MAV_CMD_DO_ENGINE_CONTROL:
```

Comment:
This might able to be moved into GCS_MIssion_Commands.cpp

Reply:
done